src/medication-request/medication-request.resource.tsx

```diff
@@ -68,7 +68,7 @@ function buildEncounterOrders(
 ): EncounterOrders {
   return {
     id: encounter?.id,
-    created: formatDate(parseDate(encounter?.period?.start)),
+    created: encounter?.period?.start,
```

Comment:
if we format the date here into a display format, then the data table sort gets messed up and the prescriptions don't get sorted in order.

Reply:
👍🏾